### PR TITLE
Add some missing events to the API docs

### LINKED ISSUE #14407: No event emitted on container rename
Service discovery tools such as [dnsdock](https://github.com/tonistiigi/dnsdock) rely on container events to remain in sync with container metadata. However, the `POST /containers/(id)/rename` API call does not generate any events, so a `docker rename` command will not notify the service discovery tool to update its metadata cache. See tonistiigi/dnsdock#22.

### PR DESCRIPTION
**Pull request by duglin**
Closes #14407

Signed-off-by: Doug Davis dug@us.ibm.com

### CODE REVIEW

File: docs/reference/api/docker_remote_api_v1.20.md [hunk 1]
```diff
@@ -1721,11 +1721,11 @@ polling (using since).
 
 Docker containers report the following events:
 
-    create, destroy, die, exec_create, exec_start, export, kill, oom, pause, restart, start, stop, unpause
+    attach, commit, copy, create, destroy, die, exec_create, exec_start, export, kill, oom, pause, rename, resize, restart, start, stop, top, unpause
 
 and Docker images report:
 
-    untag, delete
+    delete, tag, untag
```
Comment (runcom) [inline docs/reference/api/docker_remote_api_v1.20.md]: there is also `push`, `pull` and `import` for images

Comment (duglin) [inline docs/reference/api/docker_remote_api_v1.20.md]: Good catch - my grep missed a couple  :-(

**Comment by thaJeztah** [comment]
Thanks @duglin!

LGTM

Do we know which version these were added in (so that we can fix the older API versions?)

**Comment by duglin** [comment]
It was part of: https://github.com/docker/docker/pull/13443  which was merged on June 1st. Don't recall if it was cherry-picked for 1.7 or not, doubt it since @kgutwin was using 1.7 and it didn't show the event.

**Comment by thaJeztah** [comment]
Thanks Doug, I sometimes loose track of time and versions :smile:

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by moxiegirl** [comment]
Thanks @duglin LGTM

**Pull request merged**